README.md

```diff
@@ -227,11 +227,16 @@ There are two code quality analysis tools that we regularly run, spotbugs and ch
 Checkstyle enforces a consistent coding style in Kafka.
 You can run checkstyle using:
 
-    ./gradlew checkstyleMain checkstyleTest
+    ./gradlew checkstyleMain checkstyleTest spotlessCheck
 
 The checkstyle warnings will be found in `reports/checkstyle/reports/main.html` and `reports/checkstyle/reports/test.html` files in the
 subproject build directories. They are also printed to the console. The build will fail if Checkstyle fails.
 
+#### Spotless ####
+This plugin can review code by rules and can also help you check the code , it is disabled by default. Some of our code reviews use `spotless` instead of `checkstyle`:
```

Comment:
`The import orders is a part of static check. please call `spotlessApply` to optimize the imports of Java codes before filing pull request.`

Reply:
maybe we will add other check rule by spotless in future ,so i think optimize the imports maybe replaced